python-backend/app/api/mines/mine/resources/mine.py

```diff
@@ -421,9 +427,18 @@ def _mine_operation_code_processor(mine_status, index):
         return None
 
 
-def _mine_status_processor(mine_status, mine):
+def _mine_status_processor(mine_status, status_date, mine):
+
     if not mine_status:
-        return mine.mine_status
+        existing_status_date = mine.mine_status[0].status_date if mine.mine_status else None
+        if status_date is existing_status_date:
```

Comment:
I don't think this `is` reference will work as you intend. `is` evaluates that the references are the same, but `mine.mine_status[0].status_data` and `data.get('status_date')` will never be the same object.

Reply:
fixed